styx-api-service/src/test/java/com/spotify/styx/api/WorkflowResourceTest.java

```diff
@@ -537,8 +540,8 @@ public void shouldDeleteWorkflow() throws Exception {
   @Test
   public void shouldReturnErrorWhenDeleteNonexistWorkflow() throws Exception {
     sinceVersion(Api.Version.V3);
-    
-    when(storage.workflow(WORKFLOW.id())).thenReturn(Optional.empty());
+
+    doReturn(Optional.empty()).when(storage).workflow(WORKFLOW.id());
```

Comment:
Should this be `storage.workflow(WORKFLOW.id())` (as before) instead of `(storage).workflow(WORKFLOW.id()`?

Reply:
Saw it in another place as well and looked up the syntax and it seems correct, so nevermind.

Reply:
Yeah the problem here is that `storage` is a "spy", and spies sometimes don't work with the `when(...).then(...)` syntax. In this case that syntax broke when I added the `State` parameter to the `AggregateStorage` class constructor.

Reply:
👍